tests/miscutils/test_logging_config.py

```diff
@@ -15,3 +19,101 @@ def test_logging_debug(self):
 
     def test_logging_max(self):
         assert under_test.Verbosity.MAX.level() == logging.DEBUG
+
+
+class TestDefaultConfiguration:
```

Comment:
The tests added here don't seem very useful. Since what is being tested here is more configuration than logic, it seems like they will be more of an annoyance than a help. It feels like there are just being adding to hit an arbitrary threshold mark.

Reply:
Right, added these to have 90% code coverage. Should I just not check for a certain level of coverage instead?

Reply:
what is the coverage without these? We could target something like 80%?

Reply:
It's 72%-ish -- `default_logging` is a big chunk of the codebase since the repo is still pretty small. I've slimmed down the tests by just asserting that config gets called (the logic) rather than asserting how it gets called, and that brings coverage back up to 97%. What do you think?

Reply:
that works.